stable/rabbitmq/templates/statefulset.yaml

```diff
@@ -87,6 +87,11 @@ spec:
             mountPath: /opt/bitnami/rabbitmq/conf
           - name: data
             mountPath: /opt/bitnami/rabbitmq/var/lib/rabbitmq/
+          {{- if .Values.rabbitmq.loadDefinition.enabled }}
+          - name: load-definition-volume
+            mountPath: /app
```

Comment:
I see you are mounting the JSON file in `/app`, but how does rabbitmq know that it should load this specific file?

Reply:
Good question. I added an example configuration to [values.yaml](https://github.com/helm/charts/pull/11936/files#diff-6abc6beb8d3802171112b48bffcd62b0R96) and [production-values.yaml](https://github.com/helm/charts/pull/11936/files#diff-a84c43adf43a98d36ac2f66e851d1682R96) in f60624fd3f1ac6e70c5ca31e9df8672278f4484c.

Reply:
Thanks! Could you also add that info to the README? I am afraid that someone that is just following the documentation may miss that change in the extraConfiguration.

Reply:
Sure—added some extra usage instructions to README.md in a41912f8ea51a3740e8fe7e8f7ba0ebe52cdebfe.